playwright.config.ts

```diff
@@ -0,0 +1,89 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+    testDir: './e2e-tests',
+    /* Run tests in files in parallel */
+    fullyParallel: true,
+    /* Fail the build on CI if you accidentally left test.only in the source code. */
+    forbidOnly: !!process.env.CI,
+    /* Retry on CI only */
+    retries: process.env.CI ? 2 : 0,
+    /* Opt out of parallel tests on CI. */
+    workers: process.env.CI ? 1 : undefined,
+    /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+    // reporter: process.env.CI ? [['junit', { outputFile: 'results.xml' }]] : 'html',
+    /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+    use: {
+        // headless: !!process.env.CI,
+        /* Base URL to use in actions like `await page.goto('/')`. */
+        baseURL: 'https://localhost.binary.sx',
+        ignoreHTTPSErrors: true,
+
+        /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+        trace: 'on-first-retry',
+    },
+
+    /* Configure projects for major browsers */
+    projects: [
+        {
+            name: 'chromium',
+            use: { ...devices['Desktop Chrome'] },
+        },
+
+        {
+            name: 'firefox',
+            use: { ...devices['Desktop Firefox'] },
+        },
+
+        // {
+        //   name: 'webkit',
+        //   use: { ...devices['Desktop Safari'] },
+        // },
+
+        /* Test against mobile viewports. */
+        // {
+        //   name: 'Mobile Chrome',
+        //   use: { ...devices['Pixel 5'] },
+        // },
+        // {
+        //   name: 'Mobile Safari',
+        //   use: { ...devices['iPhone 12'] },
+        // },
+
+        /* Test against branded browsers. */
+        // {
+        //   name: 'Microsoft Edge',
+        //   use: { ...devices['Desktop Edge'], channel: 'msedge' },
+        // },
+        // {
+        //   name: 'Google Chrome',
+        //   use: { ...devices['Desktop Chrome'], channel: 'chrome' },
+        // },
+
+        /* Run tests for each of our packages from root of the project */
+        {
+            name: 'account',
+            testDir: './packages/account/e2e-tests',
+        },
+        {
+            name: 'integration',
+            testDir: './packages/integration/e2e-tests',
+        },
+    ],
+
+    /* Run your local dev server before starting the tests */
+    // webServer: {
```

Comment:
Does the auto start stuff actually work? Can it ever work with our setup? If not, can we just delete these lines?

Reply:
I didn't setup this because we need to serve different packages for running the tests!
For example for your example test I need to serve core and appstore

Reply:
Will remove the extra commented configs